### PR TITLE
Updating tables on tabs to match

### PR DESCRIPTION
**Pull request by tlauver**
Removed horizontal tables, added vertical property table for modify, text, and coordinate tabs.  The vertical property table is the same found on the favorites tab.

### CODE REVIEW

**Comment by tlauver** [comment]
Added fixes for Github issue #22 and #29

**Comment by csmoore** [comment]
Thanks - I think I'll get a chance to review Thurs AM

File: source/ProSymbolEditor/Models/SymbolAttributes/LabelAttributes.cs [hunk 1]
```diff
@@ -42,47 +42,36 @@ public class LabelAttributes : PropertyChangedBase
...
-                    _dateTimeValid = value;
-                }
+                _dateTimeValid = value;
 
                 NotifyPropertyChanged(() => DateTimeValid);
             }
         }
 
-        public DateTime DateTimeExpired
+        public DateTime? DateTimeExpired
```
Comment (csmoore) [inline source/ProSymbolEditor/Models/SymbolAttributes/LabelAttributes.cs]: I always forget this nullable property syntax - I guess it does have a purpose (like this one)

**Comment by tlauver** [comment]
Added fixes for Github issues #30 and #51 (we don't completely disable the panel anymore, but I did change the search bar to always appear with a white background so it doesn't look disabled)

**Comment by tlauver** [comment]
Added fixes for Github issues #60 (result count added below the search results) and #52 (will auto add the style file again if user searches and the style isn't there)

**Comment by csmoore** [comment]
Looks pretty good - a few minor issues to look at a little closer to see if they can be addressed prior to merge:

1. Missing outer scrollbar on Text tab
2. Easier nulling of  DTG: https://github.com/Esri/military-symbol-editor-addin-wpf/issues/22#issuecomment-220439163
3. Sorting countries: https://github.com/Esri/military-symbol-editor-addin-wpf/issues/29#issuecomment-220438749

**Comment by csmoore** [comment]
Merging work done for this sprint - the 3 remaining minor issues ( #66, #22, #29) have open issues to address later (next sprint).

**Pull request merged**